src/libraries/System.Collections/ref/System.Collections.cs

```diff
@@ -378,6 +378,50 @@ public partial struct Enumerator : System.Collections.Generic.IEnumerator<T>, Sy
             void System.Collections.IEnumerator.Reset() { }
         }
     }
+
```

Comment:
Nit please generate this edit with a tool: https://github.com/dotnet/runtime/blob/4d784693ebc5f91c7eede32170046355ef3969b2/docs/coding-guidelines/updating-ref-source.md